### PR TITLE
Doc: second draft of 8.1 what's new

### PR DESCRIPTION
**Pull request by pkolyvas**
**What this PR does / why we need it**:

This is an update to the what's new doc including more details about new features and links to new documentation.

This PR is focused on the updates to the Time Series panel and Geomap panel.

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v8-1.md [hunk 1]
```diff
@@ -12,47 +12,60 @@ list = false
 
 > **Note:** This topic will be updated frequently between now and the final release.
 
-This topic includes the release notes for Grafana v8.1. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+Grafana 8.1 builds upon our promise of composable, open observability platform with new panels and extends functionality launched in Grafana 8.0. We’ve got new panels, including the Geomap panel, the Annotations panel, and some great updates to the Time Series panel. For enterprise customers we’ve got additions to fine grained access control, updates to the reporting scheduler and more. We’ve also got some new transformations, updates to data sources and more. Details below.
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-1.md:15]:
```suggestion
Grafana 8.1 builds upon our promise of a composable, open observability platform with new panels and extends functionality launched in Grafana 8.0. We’ve got new panels, including the Geomap panel, the Annotations panel, and some great updates to the Time Series panel that was introduced in the 8.0 release. We’ve also got some new transformations, and updates to data sources. We’ve got additions to fine grained access control, updates to the reporting scheduler, and more for our enterprise customers. Details below.
```

File: docs/sources/whatsnew/whats-new-in-v8-1.md [hunk 2]
```diff
@@ -12,47 +12,60 @@ list = false
...
+
+We’ve summarized what’s new in the release here, but you might also be interested in the announcement blog post as well. If you’d like all the details you can checkout the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-1.md:17]:
```suggestion
In addition to what we have summarized here, you might also be interested in the announcement blog post. For all the details, check out the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Comment (achatterjee-grafana) [reply docs/sources/whatsnew/whats-new-in-v8-1.md:17]: I am really happy to see this introduction! It sets up the readers expectations nicely

File: docs/sources/whatsnew/whats-new-in-v8-1.md [hunk 3]
```diff
@@ -12,47 +12,60 @@ list = false
...
+
 
 ## Grafana OSS features
 
 These features are included in the Grafana open source edition.
 
 ### Time series panel updates
```
Comment (achatterjee-grafana) [inline docs/sources/whatsnew/whats-new-in-v8-1.md:24]: We should move this down to after Geomap and Annonations panel sections.

**Review by achatterjee-grafana — COMMENTED** [review]
Added some comments and suggestions. 

Will clone the repo and add the rest of the changes from google doc. Thanks for starting this PR.

File: docs/sources/whatsnew/whats-new-in-v8-1.md (hunk 2 above)
Comment (pkolyvas) [reply docs/sources/whatsnew/whats-new-in-v8-1.md:17]: It was all your idea - thank you!

File: docs/sources/whatsnew/whats-new-in-v8-1.md (hunk 3 above)
Comment (pkolyvas) [reply docs/sources/whatsnew/whats-new-in-v8-1.md:24]: Good point, are you willing to do that?

Comment (achatterjee-grafana) [reply docs/sources/whatsnew/whats-new-in-v8-1.md:24]: Sure

**Review by achatterjee-grafana — APPROVED** [review]
Updated markdown file with most of the information from Petro's google doc. Some questions/clarifications remain. We can update this topic with them in a subsequent PR.

**Pull request merged**